ext/ModelAnalyzerJuMPExt/ModelAnalyzerJuMPExt.jl

```diff
@@ -5,15 +5,15 @@
 
 module ModelAnalyzerJuMPExt
 
-import ModelAnalyzer
 import JuMP
 import MathOptInterface as MOI
+import ModelAnalyzer
 
 function ModelAnalyzer.analyze(
-    analyzer::T,
+    analyzer::ModelAnalyzer.AbstractAnalyzer,
     model::JuMP.GenericModel;
     kwargs...,
-) where {T<:ModelAnalyzer.AbstractAnalyzer}
```

Comment:
Don't use `where` unless:
 * it is needed in the body of the function
 * its needed across multiple input arguments
 * it's hard/impossible to write the method without it